lupa-core/src/main/kotlin/org/jetbrains/research/lupa/kotlinAnalysis/util/JupyterInverter.kt

```diff
@@ -0,0 +1,144 @@
+package org.jetbrains.research.lupa
+
+import org.json.JSONArray
+import org.json.JSONObject
+import java.io.File
+import java.io.IOException
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.util.*
+
```

Comment:
Please, lets separate this big class into several files according to their goals:
1) JupyterDatasetTransformer - a class that receives input and output paths for a dataset, and can transform dataset into python scripts
2) Data (or simple) class for storing and filtering Notebook cells
3) Utils to work with file system
4) an enum class to store cells JSON fields 

Also, all base entities (1 and 2 classes in this case) should have KDoc with their description and arguments, as well as the main functions should have descriptions too. You can find examples of KDoc in other files in the core module.